package.json

```diff
@@ -7,9 +7,12 @@
     "lint:fix": "eslint --fix . --ext .ts",
     "solidity-coverage": "NO_GAS_ENFORCE=1 npx hardhat coverage",
     "slither": "slither contracts --solc-remaps @openzeppelin=node_modules/@openzeppelin",
-    "size-contracts": "npx hardhat size-contracts"
+    "size-contracts": "npx hardhat size-contracts",
+    "install-echidna": "brew install echidna",
+    "echidna": "echidna --config echidna.yaml --contract"
```

Comment:
This does not work for me, maybe it requires an extra parameter specifying the contract?

Reply:
found out issue and fixed